x-pack/plugins/security_solution/common/experimental_features.ts

```diff
@@ -81,11 +81,6 @@ export const allowedExperimentalValues = Object.freeze({
    */
   securityFlyoutEnabled: false,
 
-  /**
-   * Enables the Elastic AI Assistant
-   */
-  assistantEnabled: false,
```

Comment:
thanks @stephmilovic for removing the experimental `assistantEnabled` feature flag